Index: src/com/helloworld/Main.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>package com.helloworld;\r\n\r\npublic class Main {\r\n\r\n    public static void main(String[] args) {\r\n        // write your code here\r\n        System.out.println(\"Hello world! test merge\");\r\n        System.out.println(\"master branch\");\r\n        System.out.println(sum(1,2,3));\r\n    }\r\n\r\n    public static int sum(int a, int b, int c) {\r\n        return a + b + c;\r\n    }\r\n}\r\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- src/com/helloworld/Main.java	(date 1532950777000)
+++ src/com/helloworld/Main.java	(date 1532951548596)
@@ -7,6 +7,7 @@
         System.out.println("Hello world! test merge");
         System.out.println("master branch");
         System.out.println(sum(1,2,3));
+        System.out.println(Math.sqrt(2));
     }
 
     public static int sum(int a, int b, int c) {
